apps/hash-api/src/graphql/resolvers/integrations/linear/sync-workspaces-with-teams.ts

```diff
@@ -91,13 +90,12 @@ export const syncLinearIntegrationWithWorkspacesMutation: ResolverFn<
       }),
     ),
     ...syncWithWorkspaces.map(async ({ workspaceEntityId, linearTeamIds }) => {
-      const workspaceAccountId = extractEntityUuidFromEntityId(
-        workspaceEntityId,
-      ) as Uuid as AccountId;
+      const workspaceOwnedById =
+        extractOwnedByIdFromEntityId(workspaceEntityId);
```

Comment:
This will return the system user's uuid, because the system user owns the user/org entity – it should remain `extractEntityUuidFromEntityId`

Reply:
I think this is highly confusing that the workspace _is_ a user or an org. A workspace _belongs_ to a user or an org, but it shouldn't _be_ the user or org. I fixed this in 58a4e240a0e58e1f11ffd11def3775b03f766e6c. Do we want to rename `workspaceOwnedById` to `workspaceOwnerId` (or `webOwnerId`)?

Reply:
Can do, sure – I'd also rename `workspaceEntityId` to `workspaceOwnerEntityId`

Reply:
Might we want to change the `EntityId` types for users and orgs to `AccountEntityId` and `AccountGroupEntityid` where possible so we have more type safety in those locations?

Reply:
We can do although I wouldn't do it in this PR because it'll add a fair bit to the diff

Reply:
Yes, I would do that (and the renaming) in a follow-up.

Reply:
I created H-776 and H-777.